GitHubReactions.ps1

```diff
@@ -0,0 +1,493 @@
+        [Alias('RepositoryUrl')]
+        [string] $Uri,
+
+        [Parameter(Mandatory, ParameterSetName='Elements', ValueFromPipelineByPropertyName)]
+        [Parameter(Mandatory, ParameterSetName='Uri', ValueFromPipelineByPropertyName)]
+        [Alias('Issue')]
+        [Alias('PullRequest')]
+        [Alias('PullRequestNumber')]
+        [int64] $IssueNumber,
+
+        [Parameter(Mandatory, ValueFromPipelineByPropertyName, ValueFromPipeline)]
+        [int64] $ReactionId,
+
+        [Parameter()]
+        [switch] $Force,
+
+        [string] $AccessToken,
+
+        [switch] $NoStatus
+    )
+
+    Write-InvocationLog
+
+    $elements = Resolve-RepositoryElements
+    $OwnerName = $elements.ownerName
+    $RepositoryName = $elements.repositoryName
+
+    $telemetryProperties = @{
+        'OwnerName' = (Get-PiiSafeString -PlainText $OwnerName)
+        'RepositoryName' = (Get-PiiSafeString -PlainText $RepositoryName)
+    }
+
+    $uriFragment = "/repos/$OwnerName/$RepositoryName/issues/$IssueNumber/reactions/$ReactionId"
+    $description = "Removing reaction $ReactionId for Issue $IssueNumber in $RepositoryName"
+
+    if ($Force.IsPresent -or $PSCmdlet.ShouldProcess($ReactionId, "Removing reaction for Issue $IssueNumber in $RepositoryName"))
+    {
+        $params = @{
+            'UriFragment' = $uriFragment
+            'Description' =  $description
+            'Method' = 'Delete'
+            'AcceptHeader' = $script:squirrelGirlAcceptHeader
+            'AccessToken' = $AccessToken
+            'TelemetryEventName' = $MyInvocation.MyCommand.Name
+            'TelemetryProperties' = $telemetryProperties
+            'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus)
+        }
+
+        return Invoke-GHRestMethod @params
+    }
+}
+
+filter Add-GitHubReactionAdditionalProperties
+{
+<#
+    .SYNOPSIS
+        Adds type name and additional properties to ease pipelining to GitHub Reaction objects.
+    .PARAMETER InputObject
+        The GitHub object to add additional properties to.
+    .PARAMETER TypeName
+        The type that should be assigned to the object.
+    .INPUTS
+        [PSCustomObject]
+    .OUTPUTS
+        GitHub.Reaction
+#>
+    [CmdletBinding()]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseSingularNouns", "", Justification="Internal helper that is definitely adding more than one property.")]
+    param(
+        [Parameter(
+            Mandatory,
+            ValueFromPipeline)]
+        [AllowNull()]
+        [AllowEmptyCollection()]
+        [PSCustomObject[]] $InputObject,
+
+        [Parameter(Mandatory)]
+        [string] $OwnerName,
+
+        [Parameter(Mandatory)]
+        [string] $RepositoryName,
+
+        [Parameter(Mandatory)]
+        [string] $IssueNumber,
+
+        [ValidateNotNullOrEmpty()]
+        [string] $TypeName = $script:GitHubReactionTypeName
+    )
+
+    if ($null -eq $InputObject) {
```

Comment:
This check is unnecessary and can be removed (it also doesn't follow style guidelines (brackets should be on a separate line).